packages/sn-client-utils/src/logger/abstract-logger.ts

```diff
@@ -0,0 +1,234 @@
+import { LeveledLogEntryWithoutScope, LogEntryWithoutScope, Logger } from '../index'
+import { LeveledLogEntry, LogEntry, LogLevel } from './log-entries'
+
+/**
+
+ * Default scope key for the Abstract Logger
+
+ */
+
+export const AbstractLoggerScope = '@sensenet/client-utils/AbstractLogger'
+
+/**
+
+ * Abstract logger instance
+
+ */
+
+export abstract class AbstractLogger implements Logger {
+  /**
+  
+     * Adds a new log entry to the logger
+  
+     * @param entry The Log entry object
+  
+     */
+  protected constructor() {
```

Comment:
Can you please remove this and the useless constructors as well from the implementations?